platform/src/error_code.rs

```diff
@@ -1,5 +1,9 @@
-/// An error code returned by the kernel.
+use core::{convert::TryFrom, fmt, mem::transmute};
+
 // TODO: Add a ufmt debug implementation for process binaries to use.
+/// A [TRD 104 error code][error-codes] returned by the kernel.
+/// 
+/// [error-codes]: https://github.com/tock/tock/blob/master/doc/reference/trd104-syscalls.md#33-error-codes
```

Comment:
This comment wording has the same problem as my original comment, in that it implies all `ErrorCode` values are generated by the kernel. `ErrorCode` values can actually come from other sources, such as error checking within `libtock-rs` APIs. I suggest something like:

```rust
/// An error code the Tock kernel may return, as specified in [TRD 104][error-codes].
/// Note that BADRVAL is not included, as it cannot be produced by the Tock kernel.
```

Reply:
Done.